examples/tensorrt_llm/common/parser.py

```
@@ -22,6 +22,7 @@
 import yaml
 from tensorrt_llm._torch.pyexecutor.config import PyTorchConfig
 from tensorrt_llm.llmapi import KvCacheConfig
+from tensorrt_llm.llmapi.llm_args import DecodingBaseConfig
```

Pylint (3.3.7)

[error] Unable to import 'tensorrt_llm.llmapi.llm_args'

(E0401)